tape/src/test/java/com/squareup/tape2/QueueFileLoadingTest.java

```diff
@@ -78,12 +76,13 @@ public void testAddWithReadOnlyFileMissesMonitor() throws Exception {
 
     // Should throw an exception.
     FileObjectQueue<String> queue =
-        new FileObjectQueue<String>(qf, new FileObjectQueue.Converter<String>() {
+        new FileObjectQueue<>(qf, new FileObjectQueue.Converter<String>() {
           @Override public String from(byte[] bytes) throws IOException {
```

Comment:
from and toStream only produce and accept nullable types if the generic type is nullable, so it's not needed there.